src/pages/landing/jump-indices/_dHero.js

```diff
@@ -0,0 +1,174 @@
+import React, { createRef, useEffect } from 'react'
+import lottie from 'lottie-web'
+import styled from 'styled-components'
+import { Container, Flex } from 'components/containers'
+import { Header } from 'components/elements'
+import { localize } from 'components/localization'
+import { LinkButton } from 'components/form'
+import device from 'themes/device.js'
+import GridSVG from 'images/svg/jump-indices/jump-indices-grid.svg'
+import JumpIndicesAnimation from 'lotties/JumpIndicesChart.json'
+
+const MainWrapper = styled(Flex)`
+    background: linear-gradient(120.16deg, #0e0e0e 34.11%, #003b81 115.35%);
```

Comment:
```suggestion
    background: linear-gradient(120.16deg, hsla(var(--color-black), 0.3411), #003b81 115.35%);
```
if this works, can replace `#003b81` hex with a color variable in `variables.js`

Reply:
It is not working if we use hsla because it is to set the opacity for the color, the percentage in `linear-gradient` above is to set the length for black color.

I did added the new variable for `#003b81` as `--color-blue-8`